src/app/components/create-offer/create-offer.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { OffersService } from './../../shared/services/offers.service';
+import { Offer } from './../../shared/model/offer.model';
+
+@Component({
+  selector: 'app-create-offer',
+  templateUrl: './create-offer.component.html',
+  styleUrls: ['./create-offer.component.css']
+})
+export class CreateOfferComponent implements OnInit {
+  offer: Offer: Array<offers>;
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+}
```

Comment:
Add function that sends offer to the service and uses the offer create route:

onSubmitOffer(){
  this.offerService.create(this.offer)
    .subscribe(........
}

Reply:
this.offer = new Offer() --> Resets the form